sdcm/cluster.py

```diff
@@ -835,6 +835,11 @@ def _wait_private_ip(self):
     def _refresh_instance_state(self):
         raise NotImplementedError()
 
+    @property
+    def cql_ip_address(self):
```

Comment:
Need to cover following cases:
- broadcast_rpc_address is 0.0.0.0
- broadcast_rpc_address is None

Reply:
1. if `Ec2MultiRegionSnitch`, than we should grab public ignoring what `broadcast_rpc_address` is in the file. https://docs.scylladb.com/operating-scylla/system-configuration/snitch/#ec2multiregionsnitch
2. if it is empty, pick up `rpc_address`
3. 0.0.0.0 is not valid configuration, cluster won't run with it